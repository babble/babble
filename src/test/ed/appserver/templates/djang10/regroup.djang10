{% regroup regroup_list by prop1 as grouped %}

{% for group in grouped %}
    {{ group.grouper }}
    
    {% for item in group.list %}
        {{ item.prop2 }}
    {% endfor %}
{% endfor %} 