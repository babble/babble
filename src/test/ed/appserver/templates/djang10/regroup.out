group#0
group#0obj#0
group#0obj#1
group#0obj#2

group#1
group#1obj#0
group#1obj#1
group#1obj#2

group#2
group#2obj#0
group#2obj#1
group#2obj#2

group#3
group#3obj#0
group#3obj#1
group#3obj#2